framework/src/media/audio/audio_manager.c

```diff
@@ -1253,58 +1256,50 @@ uint32_t get_output_bytes_to_frame(unsigned int bytes)
 
 audio_manager_result_t get_max_audio_volume(uint8_t *volume)
 {
-	// ToDo: Add logics to consider 'decibel'.
```

Comment:
I thought I remained comment here, but maybe system fault.
Media Recorder check this state already but it is different layer, so we'd better keep this.

Reply:
Reverted.